Update create-dns-records-at-godaddy.md (#447)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/dns/create-dns-records-at-godaddy.md

> [!NOTE]
> This record is used only to verify that you own your domain; it doesn't affect anything else. You can delete it later, if you like. 
  

> Follow the steps below .

Reviewer: ```suggestion
Follow the steps below.
```

  
1. To get started, go to your domains page at GoDaddy by using [this link](https://account.godaddy.com/products/?go_redirect=disabled). You'll be prompted to log in.
    
    ![GoDaddy-BP-Configure-1-1](../media/d6833ec7-9904-43fd-a877-7c663e5f5c25.png)
  
2. Under **Domains**, choose DNS under the domain that you want to edit.
    
    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
   

> 3. Choose **Add**,

Reviewer: ```suggestion
3. Choose **Add**.
```

    
    ![GoDaddy-BP-Configure-1-4](https://user-images.githubusercontent.com/45987684/56527673-ffb3b300-651b-11e9-91c2-83dc9fe5ca30.PNG)
  

> 4. Choose **TXT (Text)** from the drop-down list, In the boxes for the new record, type or copy and paste the values from the following table.

Reviewer: ```suggestion
4. Choose **TXT (Text)** from the drop-down list. In the boxes for the new record, type or copy and paste the values from the following table.
```

## Add an MX record so email for your domain will come to Office 365
<a name="BKMK_add_MX"> </a>

> Follow the steps below 

Reviewer: ```suggestion
Follow the steps below.
```

  
1. To get started, go to your domains page at GoDaddy by using [this link](https://account.godaddy.com/products/?go_redirect=disabled). You'll be prompted to log in.
    
    ![GoDaddy-BP-Configure-1-1](../media/d6833ec7-9904-43fd-a877-7c663e5f5c25.png)
  
2. Under **Domains**, choose DNS under the domain that you want to edit.
    
    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
   

> 3. Choose **Add**,

Reviewer: ```suggestion
3. Choose **Add**.
```

    
    ![GoDaddy-BP-Configure-1-4](https://user-images.githubusercontent.com/45987684/56527673-ffb3b300-651b-11e9-91c2-83dc9fe5ca30.PNG)
  
4. Choose **MX (Mail Exchanger)** from the drop-down list. 
    
    ![GoDaddy-BP-Configure-2-0](https://user-images.githubusercontent.com/45987684/56528642-85842e00-651d-11e9-8dd8-217f468f9a18.PNG)
  
5. In the boxes for the new record, type or copy and paste the values from the following table.
    
    (Select the **TTL** value from the drop-down list.) 
    
    |**Record type**|**Host**|**Points to**|**Priority**|**TTL**|
    |:-----|:-----|:-----|:-----|:-----|
    |MX (Mail Exchanger)  <br/> |@  <br/> | *\<domain-key\>*  .mail.protection.outlook.com  <br/> **Note:** Get your  *\<domain-key\>*  from your Office 365 portal account.           [How do I find this?](../get-help-with-domains/information-for-dns-records.md)          |10  <br/> For more information about priority, see [What is MX priority?](https://support.office.com/article/2784cc4d-95be-443d-b5f7-bb5dd867ba83.aspx) <br/> |1 hour  <br/> |
   
6. Choose **Save**.

  
## Add the CNAME records that are required for Office 365
<a name="BKMK_add_CNAME"> </a>

> Follow the steps below 

Reviewer: ```suggestion
Follow the steps below .
```

  
1. To get started, go to your domains page at GoDaddy by using [this link](https://account.godaddy.com/products/?go_redirect=disabled). You'll be prompted to log in.
    
    ![GoDaddy-BP-Configure-1-1](../media/d6833ec7-9904-43fd-a877-7c663e5f5c25.png)
  
2. Under **Domains**, choose DNS under the domain that you want to edit.
    
    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
   

> 3. Choose **Add**,

Reviewer: ```suggestion
3. Choose **Add**.
```

    |CNAME (Alias)  <br/> |lyncdiscover  <br/> |webdir.online.lync.com  <br/> |1 hour  <br/> |
    |CNAME (Alias)  <br/> |msoid  <br/> |clientconfig.microsoftonline-p.net  <br/> |1 hour  <br/> |
    |CNAME (Alias)  <br/> |enterpriseregistration  <br/> |enterpriseregistration.windows.net  <br/> |1 hour  <br/> |
    |CNAME (Alias)  <br/> |enterpriseenrollment  <br/> |enterpriseenrollment.manage.microsoft.com  <br/> |1 hour  <br/> |
   

  

> 7. Repeat these step to add the CNAME record until you have created all six of the CNAME records.

Reviewer: ```suggestion
6. Repeat these steps to add the next CNAME record until you have created all six of the CNAME records.
```

    

  
## Add a TXT record for SPF to help prevent email spam
<a name="BKMK_add_TXT"> </a>

> [!IMPORTANT]
> You cannot have more than one TXT record for SPF for a domain. If your domain has more than one SPF record, you'll get email errors, as well as delivery and spam classification issues. If you already have an SPF record for your domain, don't create a new one for Office 365. Instead, add the required Office 365 values to the current record so that you have a  *single*  SPF record that includes both sets of values. 
  

> Follow the steps below 

Reviewer: ```suggestion
Follow the steps below.
```

  
1. To get started, go to your domains page at GoDaddy by using [this link](https://account.godaddy.com/products/?go_redirect=disabled). You'll be prompted to log in.
    
    ![GoDaddy-BP-Configure-1-1](../media/d6833ec7-9904-43fd-a877-7c663e5f5c25.png)
  
2. Under **Domains**, choose DNS under the domain that you want to edit.
    
    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
   

> 3. Choose **Add**,

Reviewer: ```suggestion
3. Choose **Add**.
```

   
    ![GoDaddy-BP-Configure-4-1](../media/7c724f02-c9b3-42ab-b9c0-78959fa6ffad.png)
  
6. Choose **Save**.
    
  
## Add the two SRV records that are required for Office 365
<a name="BKMK_add_SRV"> </a>

> Follow the steps below 

Reviewer: ```suggestion
Follow the steps below.
```

  
1. To get started, go to your domains page at GoDaddy by using [this link](https://account.godaddy.com/products/?go_redirect=disabled). You'll be prompted to log in.
    
    ![GoDaddy-BP-Configure-1-1](../media/d6833ec7-9904-43fd-a877-7c663e5f5c25.png)
  
2. Under **Domains**, choose DNS under the domain that you want to edit.
    
    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
   

> 3. Choose **Add**,

Reviewer: ```suggestion
3. Choose **Add**.
```
